### PR TITLE
Added some documentation for Slider

### PR DESCRIPTION
**Pull request by KoBeWi**
Yesterday I was doing something with sliders and noticed they don't have documentation and it bothered me. So here it is.

Also fixed a mistake in VSlider.

### CODE REVIEW

File: doc/classes/Slider.xml [hunk 1]
```diff
@@ -14,14 +14,18 @@
 	</methods>
 	<members>
 		<member name="editable" type="bool" setter="set_editable" getter="is_editable">
+		If true, slider can be interacted with. Value can be changed only by code if slider is not editable.
```
Comment (Calinou) [inline doc/classes/Slider.xml]:
```suggestion
		If [code]true[/code], the slider can be interacted with. If [code]false[/code], the value can be changed only by code.
```

File: doc/classes/Slider.xml [hunk 2]
```diff
@@ -14,14 +14,18 @@
...
 		</member>
 		<member name="focus_mode" type="int" setter="set_focus_mode" getter="get_focus_mode" enum="Control.FocusMode">
 		</member>
 		<member name="scrollable" type="bool" setter="set_scrollable" getter="is_scrollable">
+		If true, value can be changed using mouse wheel.
```
Comment (Calinou) [inline doc/classes/Slider.xml]:
```suggestion
		If [code]true[/code], the value can be changed using the mouse wheel.
```

File: doc/classes/Slider.xml [hunk 3]
```diff
@@ -14,14 +14,18 @@
...
 		</member>
 		<member name="tick_count" type="int" setter="set_ticks" getter="get_ticks">
+		Number of ticks displayed on the slider, including border ticks. Ticks are uniformly distributed value markers.
```
Comment (Calinou) [inline doc/classes/Slider.xml]:
```suggestion
		Number of ticks displayed on the slider, including border ticks. Ticks are uniformly-distributed value markers.
```

File: doc/classes/Slider.xml [hunk 4]
```diff
@@ -14,14 +14,18 @@
...
 		</member>
 		<member name="ticks_on_borders" type="bool" setter="set_ticks_on_borders" getter="get_ticks_on_borders">
+		If true, slider will display ticks for minimum and maximum value.
```
Comment (Calinou) [inline doc/classes/Slider.xml]:
```suggestion
		If [code]true[/code], the slider will display ticks for minimum and maximum values.
```

**Review by Calinou — COMMENTED** [review]
Thanks for contributing to the documentation :slightly_smiling_face: Please take a look at the suggestions I made. See [Docs writing guidelines](http://docs.godotengine.org/en/latest/community/contributing/docs_writing_guidelines.html) for future reference.

**Comment by KoBeWi** [comment]
Thanks, I applied the suggestions.

btw, I noticed that in some docs, `if true` is actually "if true", not "if [code]true[/code]" like you suggested. This probably should be made more consistent.

**Comment by mhilbrunner** [comment]
Thanks for helping out with the docs!

>btw, I noticed that in some docs, if true is actually "if true", not "if [code]true[/code]" like you suggested. This probably should be made more consistent.

Quite right :) If you want to, more PRs are always welcome!

**Review by mhilbrunner — APPROVED** [review]

**Comment by akien-mga** [comment]
Cherry-picked for 3.1.1.

**Pull request merged**